### ISSUE TITLE
concept weighting support

### ISSUE THREAD

**Issue by FodT** [MediaMath/t1-python#86]
doesn't look like strategy_concepts supports POSTing directly with weighting (i could be wrong on this), but POSTing to `/strategies/:id/concepts` allows it: 

```
concepts.1.id:828276
concepts.1.weighting:BUDGET
concepts.1.percent:30.00
concepts.2.id:828032
concepts.2.weighting:BUDGET
concepts.2.percent:30.00
concepts.3.id:828020
concepts.3.weighting:BUDGET
concepts.3.percent:40.00
```

**Comment by pswaminathan** [comment]
I'd be tempted to do it like how we have other strategy children [implemented](https://github.com/MediaMath/t1-python/blob/master/terminalone/models/strategy.py#L151-174)—kind of deferred to the consumer.

**Comment by missinnale** [comment]
Working on UI-Automation framework and integrating this service in to speed up tests. Having weighting support would be really useful, let me know any updates on this. Thanks!

**Comment by missinnale** [comment]
Not sure if this is already known but when retrieving a strategy_concept object the weighting and percent of that weighting are included in the object:

```bash
StrategyConcept(_type='strategy_concept', percent='100.00', updated_on=datetime.datetime(2016, 8, 23, 15, 40, 10, tzinfo=FixedOffset(0)), status=True, version=0, id=3878832, weighting='BUDGET', concept_id=875146, name='Strategy Concepts #3878832', created_on=datetime.datetime(2016, 8, 23, 15, 40, 10, tzinfo=FixedOffset(0)), strategy_id=1503600)
```

I'd be interested to know what happens if in the StrategyConcept model under `_pull` if the keys
```
'weighting': 'VALUE'
'percent': '###'
```
were added.

Would that allow them to pass through? Currently when I try to add those values it doesn't work, but I can't tell if thats because they aren't explicitly stated as part of the model or the endpoint is rejecting them (guessing based on what @FodT stated it would be the model). Considering I haven't worked on this service I could be completely wrong and any knowledge that could be thrown my way would be helpful.

**Comment by FodT** [comment]
Solved by #92
